neonvm/runner/main.go

```diff
@@ -558,6 +583,22 @@ func runInitScript(logger *zap.Logger, script string) error {
 	return nil
 }
 
+func qemuDiskArgs(id string, file string, extraOptions string, deviceExtraOptions string, diskIdxCnt *int) []string {
+	if extraOptions != "" && extraOptions[0] != ',' {
+		extraOptions = "," + extraOptions
+	}
+	if deviceExtraOptions != "" && deviceExtraOptions[0] != ',' {
+		deviceExtraOptions = "," + deviceExtraOptions
+	}
+
+	args := []string{}
+	args = append(args, "-drive", fmt.Sprintf("id=%s,file=%s,if=none,index=%d%s", id, file, *diskIdxCnt, extraOptions))
```

Comment:
maybe, we shoud use `-blockdev` here, if we have `-device` also? 

See also https://github.com/neondatabase/autoscaling/pull/822#issuecomment-1986842294

Reply:
i'll have a try!